frontera/contrib/backends/sqlalchemy/components.py

```diff
@@ -173,10 +173,10 @@ def get_next_requests(self, max_n_requests, partition_id, **kwargs):
         try:
             for item in self._order_by(self.session.query(self.queue_model).filter_by(partition_id=partition_id)).\
                     limit(max_n_requests):
-                method = item.method or 'GET'
+                method = item.method or b'GET'
```

Comment:
What type item.method will get? In case of sqlite and postgresql? I thought we agreed on native_str before SQLA and bytes anywhere else in Frontera.